developer/ruby/Brewfile

```diff
@@ -1,3 +1,6 @@
+# Taps
+tap 'themainingredient/homebrew-qt5.5'
```

Comment:
Renamed to `themainingredient/homebrew-tmi`.

Reply:
done!